dts/bindings/spi/nxp,lpc-spi.yaml

```diff
@@ -5,11 +5,4 @@ description: NXP LPC SPI controller
 
 compatible: "nxp,lpc-spi"
 
-include: spi-controller.yaml
-
-properties:
-    reg:
-      required: true
-
-    interrupts:
-      required: true
+include: [spi-controller.yaml, "nxp,lpc-flexcomm.yaml"]
```

Comment:
remove the quotes around "nxp,lpc-flexcomm.yaml"

Reply:
I get a device tree error without the quotes:
```
-- west build: making build dir /home/maureen/zephyrproject/zephyr/samples/sensor/fxos8700/build pristine
-- west build: generating a build system
Including boilerplate (Zephyr base): /home/maureen/zephyrproject/zephyr/cmake/app/boilerplate.cmake
-- Application: /home/maureen/zephyrproject/zephyr/samples/sensor/fxos8700
-- Zephyr version: 2.2.99 (/home/maureen/zephyrproject/zephyr)
-- Found PythonInterp: /usr/bin/python3.6 (found suitable version "3.6.9", minimum required is "3.6") 
-- Board: lpcxpresso55s69_cpu0
-- Found west: /home/maureen/.local/bin/west (found suitable version "0.7.2", minimum required is "0.7.1")
-- Found toolchain: zephyr (/home/maureen/tools/zephyr-sdk/0.11.1)
-- Found BOARD.dts: /home/maureen/zephyrproject/zephyr/boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts
devicetree error: 'nxp' not found
CMake Error at /home/maureen/zephyrproject/zephyr/cmake/dts.cmake:229 (message):
  gen_defines.py failed with return code: 1
Call Stack (most recent call first):
  /home/maureen/zephyrproject/zephyr/cmake/app/boilerplate.cmake:503 (include)
  /home/maureen/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:24 (include)
  /home/maureen/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:35 (include_boilerplate)
  CMakeLists.txt:4 (find_package)


-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /home/maureen/.local/bin/cmake -B/home/maureen/zephyrproject/zephyr/samples/sensor/fxos8700/build -S/home/maureen/zephyrproject/zephyr/samples/sensor/fxos8700 -GNinja -DBOARD=lpcxpresso55s69_cpu0 -DCONF_FILE=prj_accel.conf
```

Reply:
guess this is due to the comma in the name.